modules/virt-creating-hpp-basic-storage-pool.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-hpp-basic-storage-pool_{context}"]
+= Creating a hostpath provisioner with a basic storage pool
+
+You configure a hostpath provisioner (HPP) with a basic storage pool by creating an HPP custom resource (CR) with a `storagePools` stanza. The storage pool specifies the name and path used by the CSI driver.
+
+.Prerequisites
+
+* The directories specified in `spec.storagePools.path` must have read/write access.
+* The storage pools must not be in the same partition as the operating system. Otherwise, the operating system partition might be filled to capacity, which will impact performance or cause the node to become unstable or unusable.
+
+.Procedure
+
+. Create an `hpp_cr.yaml` file with a `storagePools` stanza as in the following example:
++
+[source,yaml]
+----
+apiVersion: hostpathprovisioner.kubevirt.io/v1beta1
+kind: HostPathProvisioner
+metadata:
+  name: hostpath-provisioner
+spec:
+  imagePullPolicy: IfNotPresent
+  storagePools: <1>
+  - name: any_name
+    path: "/var/myvolumes" <2>
+workload:
+  nodeSelector:
+    kubernetes.io/os: linux
+----
+<1> The `storagePools` stanza is an array to which you can add multiple entries.
+<2> Specify the storage pool directories under this node path.
+
+. Save the file and exit.
+
+. Create the HPP:
```

Comment:
Nit, and I haven't been consistently enforcing this, but the OCP docs guidelines still specify:

> If a step in a procedure is to run a command, make sure that the step text includes an explicit instruction to "run" or "enter" the command. In most cases, use one of the following patterns to introduce the code block:
> 
> \<Step description> by running the following command:
> 
> \<Step description> by entering the following command:
> 
> \<Step description>, run the following command:
> 
> \<Step description>, enter the following command:

Reply:
Sigh. Yes, I know.